src/main/resources/static/common/js/graphQLService.js

```diff
@@ -45,5 +45,12 @@ angular
                     success(response.data.data.getLocation);
                 }, error);
             }
+            //cartItems
+            instance.getCartItems = function(success, error) {
+                let requestBody = {"query": "query{getCartItems{id,userId,itemId,name,description,image,realInStock,quantity}}"}
```

Comment:
Let's change the arguments to be passed in argument. And for all places which make this query, only select fields needed to minimize payload. That's the purpose of GraphQL.